### ISSUE TITLE
Running with prediction models

### ISSUE THREAD

**Issue by ValHook** [sertalpbilal/fpl_hindsight_optimization#1]
Hi, this work is amazing!
It would be interesting to run the solver with a points predictor instead of hindsight to see how religiously following a model would have performed.
Is it something you have done, or would be curious to try?

**Comment by sertalpbilal** [comment]
Yes, it can be repurposed for that. I have tried it in the past, but haven't pushed my codes to this repo, and not sure if I can find them. If you are interested and have any questions, please let me know.

**Comment by ValHook** [comment]
Thanks for the quick reply!

I would be interested to know the results, such as ranking in terms of percentile.

And if the performance was great:
- Which model or models you used?
- How you approached the rolling horizon problem, assuming that any predictive model is limited to just a few game weeks in the future, with growing uncertainty?

**Comment by sertalpbilal** [comment]
I have used mainly FPLReview and Mikkel Tokvam's models.
Since models only give a few GWs of data, I have used a rolling horizon, but usually prevented scheduled transfers down the line. Also, giving a higher weight to closer GWs usually is a good way to have transfers early than later.

Don't think I have anything on the performance but will share if I can find it.
